v3/testdata/dNSNameNoUnderscores.pem

```diff
@@ -0,0 +1,37 @@
+Certificate:
+    Data:
+        Version: 3 (0x2)
+        Serial Number: 3 (0x3)
+        Signature Algorithm: ecdsa-with-SHA256
+        Issuer: 
+        Validity
+            Not Before: Jan  1 00:00:00 1970 GMT
+            Not After : Nov 30 00:00:00 9998 GMT
+        Subject: 
+        Subject Public Key Info:
+            Public Key Algorithm: id-ecPublicKey
+                Public-Key: (256 bit)
+                pub:
+                    04:89:6d:b1:15:7a:e5:f1:7f:6e:6c:46:d6:2e:0b:
+                    32:9a:c1:72:a8:96:a8:b0:ed:20:1f:49:64:71:2c:
+                    dc:00:f0:7d:90:47:c8:f1:f3:f4:c3:33:dd:f7:af:
+                    98:17:69:73:b2:05:35:74:75:96:4d:66:8a:6f:1a:
+                    58:b6:e0:a5:2c
+                ASN1 OID: prime256v1
+                NIST CURVE: P-256
+        X509v3 extensions:
+            X509v3 Subject Alternative Name: 
+                DNS:no.underscores.com
```

Comment:
If it's not _too_ much hassle, is it possible to put these under the `.test` or `.example` (c.f. https://www.iana.org/domains/reserved )

Reply:
No trouble! I swear, half the ideas I come up with for maintaining this repo have to do with easy test data generation.